projects/js-packages/image-guide/src/MeasurableImage.ts

```diff
@@ -11,15 +11,19 @@ export class MeasurableImage {
 	readonly node: HTMLElement | HTMLImageElement;
 	private getURLCallback: SourceCallbackFn;
 
+	public fetch = fetch;
```

Comment:
On review, this does feel a bit odd. Do you think it's worth saying `fetch = window.fetch` just to add extra context for future devs reading the code?

Reply:
Thanks, good idea.